vcpkg.json

```diff
@@ -0,0 +1,26 @@
+{
+    "name": "tiledb",
+    "version-string": "2.15.0",
+    "builtin-baseline": "f14984af3738e69f197bf0e647a8dca12de92996",
```

Comment:
Do we have to specify this both here and in the submodule's version?

Reply:
AFAIK, not technically, and this might be a belt and suspenders approach, but I think its for the best. vcpkg seems to have made a somewhat odd choice going full in on Git semantics both officially and semi-officially. Doing it this way means we don't care what the submodule is at as our reference here will either give us exactly what we want, or fail the build if (for some unknowable reason) we accidentally moved the submodule version to before this hash.